en/includes/guides/authentication/jit-user-provisioning.md

```diff
@@ -43,6 +43,12 @@ To enable/disable JIT user provisioning for an external Identity provider:
     <br />
     - When JIT is disabled, a user profile is not created. Therefore, there is no location to store the attributes of the user. In such cases, the attributes are directly passed to the application.
 
+{% if product_name == "Asgardeo" or (product_name == "WSO2 Identity Server" and is_version == "7.2.0") %}
```

Comment:
shouldn't this check be changed as 
{% if product_name == "Asgardeo" or (product_name == "WSO2 Identity Server" and is_version > "7.1.0") %}
because this will be available any version after 7.1.0 of IS?